tests/postgres_tests/test_constraints.py

```diff
@@ -856,6 +878,11 @@ def test_spgist_include_not_supported(self):
                 with self.assertRaisesMessage(NotSupportedError, msg):
                     editor.add_constraint(RangesModel, constraint)
 
+    # RemovedInDjango50Warning: When the deprecation ends, remove the
+    # @ignore_warnings and opclasses=['range_ops']. Replace the
+    # expressions argument with:
+    # expressions=[(OpClass('ints', name='range_ops'), RangeOperators.ADJACENT_TO)]
+    @ignore_warnings(category=RemovedInDjango50Warning)
```

Comment:
We should have both versions covered by tests, so I moved test methods with `opclasses` to the new class `ExclusionConstraintOpclassesDepracationTests`.